### DISCUSSION TITLE
Sqlite drivers and PRAGMA Statements

### DISCUSSION THREAD

**Discussion by askmrsinh** [mikro-orm/mikro-orm#3303] [Q&A]
How do I use `PRAGMA Statements` with sqlite3/better-sqlite-3?
Basically, I would to take advantage of [WAL](https://github.com/WiseLibs/better-sqlite3/blob/master/docs/performance.md) for my queries.

**Comment by haoxi911** [comment]
Same here, @askmrsinh did you figure out how to do it in Mikro ORM?

**Reply by askmrsinh** [reply]
No, I haven't.

**Comment by B4nan** [comment]
Pragma is a regular SQL query, you dont need any special methods, just do `em.execute('pragma ...')`.

**Comment by haoxi911** [comment]
Yes, I was able to enable it by running the query immediately after establishing connection.
```typescript
const orm = await MikroORM.init<SqliteDriver>(config);
await orm.em.getConnection().execute('PRAGMA journal_mode=WAL;');
```
